astropy/wcs/wcs.py

```diff
@@ -53,6 +53,13 @@
     else:
         _wcs = None
 
+if _wcs is not None:
+    _parsed_version = _wcs.__version__.split('.')
+    if int(_parsed_version[0]) == 5 and int(_parsed_version[1]) < 8:
+        raise ImportError(
+            "astropy.wcs is built with wcslib {0}, but only versions 5.8 and "
+            "later on the 5.x series are known to work.")
```

Comment:
Maybe have this instead be something like "only 5.8 and later (or the verion packaged with astropy)"?  I.e., make it clear that a possible "solution" is to use the builtin one if for whatever reason it's difficult to upgrade wcslib